bundles/org.openhab.core.karaf/pom.xml

```diff
@@ -49,6 +49,12 @@
       <version>${karaf.compile.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.karaf.kar</groupId>
+      <artifactId>org.apache.karaf.kar.core</artifactId>
+      <version>${karaf.tooling.version}</version>
+      <scope>compile</scope>
```

Comment:
```suggestion
      <version>${karaf.compile.version}</version>
      <scope>provided</scope>
```

Reply:
Done. I have a question: Why is the `EventPublisher` a static member? If it was an instance member we could omit the null-checks.

Reply:
I don't know of a good reason why it should be static either.
Maybe @kaikreuzer remembers why it was added like this in #81.